ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/healthcheck/HealthCheckAgent.java

```diff
@@ -49,6 +51,7 @@
   public static final String METASTORE_CHECK_NAME = "metastore";
   public static final String KAFKA_CHECK_NAME = "kafka";
   public static final String COMMAND_RUNNER_CHECK_NAME = "commandRunner";
+  public static final String QUERY_TASKS_METADATA_NAME = "metadata";
```

Comment:
Use a different name for this. Metadata in ksql will be confused with metastore.

Reply:
We actually can just remove it